rhino/src/main/java/org/mozilla/javascript/TokenStream.java

```diff
@@ -577,11 +577,11 @@ private static int stringToKeywordForES(String name, boolean isStrict) {
         return id & 0xff;
     }
 
-    @SuppressWarnings("AndroidJdkLibsChecker")
     private static boolean isValidIdentifierName(String str) {
```

Comment:
This mehtod has used https://developer.android.com/reference/java/lang/String#codePoints(), wich would be available since API 24.

I refactored this, so that we not use the String->IntStream->toArray pipeline, as I think this version might be more performant